src/components/form/Form.js

```diff
@@ -1,25 +1,57 @@
 import React, {useState} from 'react'
-import FormVenue from './FormVenue'
-import FormTime from './FormTime'
+import {useApp} from '../../context/AppContext'
 
 import './Form.css';
 
 const Form = () => {
```

Comment:
This form looks really intuitive after all the work y'all put in. Thanks for all of this!